llmfoundry/models/layers/dmoe.py

```diff
@@ -66,11 +72,12 @@ def jitter(self, x: torch.Tensor) -> torch.Tensor:
         )
         return low + noise * (high - low)
 
-    def _top_k(self, scores: torch.Tensor) -> torch.Tensor:
+    def _top_k(self, scores: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
         if self.moe_top_k == 1:
-            return scores.max(
+            values, indices = scores.max(
                 dim=-1,
             )  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
why is this pyright necessary?

Reply:
that was there before

Reply:
removed it and the one below, fixed return type